eng/native/init-compiler-and-cmake.cmd

```diff
@@ -2,6 +2,11 @@
 rem
 rem This file locates VS C++ compilers and cmake for windows.
 
+rem Make sure the current directory stays intact
+set VSCMD_START_DIR="%CD%"
+
+set __VSCMD_ARG_NO_LOGO=yes
```

Comment:
Does this one remove the version banner when initializing the MSVC environments? If so this is useful when doing investigations around regressions introduced by toolsets. Otherwise, this looks good.